config/database.yml

```diff
@@ -17,11 +17,21 @@
 development: &default
   adapter: postgresql
   encoding: unicode
-  database: champaign
+  database: champaign_test
```

Comment:
It needs to be the same as for `champaign-ak-processor`, which also has `champaign_test`.